indico/modules/vc/templates/emails/deleted.html

```diff
@@ -1,7 +1,9 @@
 {% extends 'emails/base.html' %}
 
 {% block subject -%}
-    [{{ plugin.friendly_name }}] Videoconference room deleted: {{ vc_room.name }}
+    {% autoescape false %}
```

Comment:
strip whitespace inside. this is an email subject and it might look ugly if there's surrounding whitespace (not sure if we strip it in make_email or not)

same in the other file